format-specs/geoparquet.md

```diff
@@ -50,57 +50,160 @@ Version of the geoparquet spec used, currently 0.3.0
-        MEMBER["World Geodetic System 1984 (Transit)"],
-        MEMBER["World Geodetic System 1984 (G730)"],
-        MEMBER["World Geodetic System 1984 (G873)"],
-        MEMBER["World Geodetic System 1984 (G1150)"],
-        MEMBER["World Geodetic System 1984 (G1674)"],
-        MEMBER["World Geodetic System 1984 (G1762)"],
-        MEMBER["World Geodetic System 1984 (G2139)"],
-        ELLIPSOID["WGS 84",6378137,298.257223563,
-            LENGTHUNIT["metre",1]],
-        ENSEMBLEACCURACY[2.0]],
-    PRIMEM["Greenwich",0,
-        ANGLEUNIT["degree",0.0174532925199433]],
-    CS[ellipsoidal,2],
-        AXIS["geodetic longitude (Lon)",east,
-            ORDER[1],
-            ANGLEUNIT["degree",0.0174532925199433]],
-        AXIS["geodetic latitude (Lat)",north,
-            ORDER[2],
-            ANGLEUNIT["degree",0.0174532925199433]],
-    USAGE[
-        SCOPE["Not known."],
-        AREA["World."],
-        BBOX[-90,-180,90,180]],
-    ID["OGC","CRS84"]]
-```
+
+Note: earlier versions of OGC:CRS84 used a single datum instead of a datum ensemble.  The number of elements in the datum ensemble may vary over time when new realizations of WGS 84 are added to the ensemble
+
 
 Due to the large number of CRSes available and the difficulty of implementing all of them, we expect that a number of implementations will start without support for the optional `crs` field.
-Users are recommended to store their data in longitude, latitude (OGC:CRS84 or not including the `crs` field) for it to work with the widest number of tools. But data that is better served in particular projections can choose to use an alternate coordinate reference system. We expect many tools will support alternate CRSes, but encourage users to check to ensure their chosen tool supports their chosen crs.
+Users are recommended to store their data in longitude, latitude (OGC:CRS84 or not including the `crs` field) for it to work with the widest number of tools. Data that are more appropriately represented in particular projections may use an alternate coordinate reference system. We expect many tools will support alternate CRSes, but encourage users to check to ensure their chosen tool supports their chosen CRS.
+
+
+For implementations that operate entirely with longitude, latitude coordinates
+and are not CRS-aware or do not have easy access to CRS-aware libraries that can
+fully parse PROJJSON, it may be possible to infer that coordinates conform to
+the OGC:CRS84 CRS based on elements of the `crs` field.  For simplicity, Javascript
+object dot notation is used to refer to nested elements.
+
+The CRS is likely equivalent to OGC:CRS84 for a GeoParquet file if the `id` element is present:
+
+* `id.authority` = `"OGC"` and `id.code` = `"CRS84"`
+* `id.authority` = `"EPSG"` and `id.code` = `4326` (due to longitude, latitude ordering in this specification)
+
+The CRS is likely equivalent to OGR:CRS84 if all of the following are true:
```

Comment:
@rouault I'd appreciate your thoughts on the following rules for at least the common cases (we don't need exhaustive rules here).  In particular, I don't know if the units or directions are likely to always be present or even necessary (they are missing from the OGC:CRS84 xml linked above).